tests/test.ts

```diff
@@ -82,11 +82,14 @@ describe.skip('Test clasp clone <scriptId> function', () => {
     expect(result.status).to.equal(0);
   });
   it('should give an error on a non-existing project', () => {
+    const settings = JSON.parse(fs.readFileSync('.clasp.json', 'utf8'));
+    fs.removeSync('.clasp.json');
```

Comment:
Why are we removing this file? Where was it created? We should delete the file in the same test it was created so that the tests are isolated.

Reply:
Yeah, so we're removing it because otherwise you'll get the `Project file .clasp.json already exists.` error, whereas we are really targeting the error that says `> Did you provide the correct scriptId?`. We should probably have an additional test for when a `.clasp.json` already exists.

We re-write it at the end of the test so that the following test, for `clasp pull` works. We could definitely separate things though if that's what is better.

Reply:
OK.
Can you please include a detailed comment for these points:
- In the file comment, state `.clasp.json` exists for this test suite as a global file.
- In this test, state we are temporarily removing `.clasp.json` to test `clasp clone` fails when the project doesn't have this file.

Reply:
So it's not really global, it's created as the tests go through. That's why they're in a specific order. It's definitely not ideal, each test should be independent of another, but it's what we have for now.

Reply:
To clarify, it is created in a previous test, which tests `clasp create <title>`. Then we use the `.clasp.json` from that created script in the following tests.

Reply:
OK. We'll really need to fix this later, as we can't have dependencies between test (where reordering the tests will fail).

Reply:
@grant yeah I know, it's tough though because many tests need a real `scriptId` for them to work 😞